app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/Evaluation.tsx

```diff
@@ -1155,6 +1155,12 @@ export default function Evaluation(props: EvaluationProps) {
                         colorscale: confusionMatrixConfig.log
                           ? confusionMatrix?.colorscale || "viridis"
                           : "viridis",
+                        hovertemplate:
+                          [
+                            "<b>count: %{z:d}</b>",
+                            `ground_truth: %{y}`,
```

Comment:
@lanzhenw ideally we would show the actual field names here, rather than the hard-coded `ground_truth` and `predictions` strings, like is implemented [here](https://github.com/voxel51/fiftyone/blob/8beca16fd9931f9262eef8b30e581510cef1f36b/fiftyone/core/plots/plotly.py#L137-L165).

In order to achieve that, you'd need access to `config.gt_field` and `config.pred_field` from the evaluation's config. Do you have access to those in the frontend? If you do, then let's use the actual field names.

If you don't easily have access to the field names, then please just use `ground truth` rather than `ground_truth` and we can add field names in a follow-up PR.